.travis.yml

```diff
@@ -13,12 +13,17 @@ env:
     - STACK_DATA_SLOW_TESTS=1
     - COVERALLS_PARALLEL=true
 
+before_install:
+  - pip install --upgrade coveralls setuptools>=44 setuptools_scm>=3.4.3 pep517
+
 install:
-  - pip install coveralls
-  - pip install .[tests]
+  - python3 -m pep517.build .
+  - ls -l
+  - mv ./dist/*.whl ./dist/stack_data-0.CI-py3-none-any.whl
```

Comment:
Why this complex filename?

Reply:
We cannot install a wheel by mask. So we rename it into some name we can use. Wheel name must follow certain format. So I have taken the name of a generated wheel an replaced version part in it to `0.CI`

Reply:
Wow, I'm really surprised to learn that pip refuses to install wheels with invalid names. That's very unconventional.

Can you put the filename in an environment variable so that it's not duplicated?